app/questionnaire/questionnaire_schema.py

```diff
@@ -133,8 +133,9 @@ def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
     @cached_property
-    def min_and_max_map(self) -> Any:
-        return make_immutable(self._min_and_max_map)
+    # Type ignore: safe to assume _min_and_max_map exists
```

Comment:
Type ignore is that we can assume the return type rather than existence, but otherwise looks good